build.gradle

```diff
@@ -66,6 +66,7 @@ ext {
     commonsLang: 'org.apache.commons:commons-lang3:3.4',
     commonsCli: 'commons-cli:commons-cli:1.3',
     snakeyaml: 'org.yaml:snakeyaml:1.16',
+    commonmark: 'com.atlassian.commonmark:commonmark:0.9.0',
```

Comment:
Commonmark is licensed under BSD 2 clause. I'm pretty sure we're OK to use it, but do we need to replicate the BSD notice or get approval, etc?